ui-components/src/page_components/listing/listing_sidebar/SidebarBlock.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react"
+import { Heading, HeaderType } from "../../../headers/Heading"
+
+export interface SidebarBlockProps {
+  children: React.ReactNode
+  className?: string
+  title?: string
+  titlePriority?: number
```

Comment:
The `titlePriority` prop is used for the `Heading` component purposes, shouldn't we pick a type from the `Heading` component?

Reply:
The Heading component doesn't type that prop, so I just extended the whole interface 👍